UnityProject/Assets/Scripts/MapSaver/MapLoader.cs

```diff
@@ -310,7 +310,8 @@ public static void ProcessIndividualObject(MapSaver.CompactObjectMapData Compact
 
 			if (string.IsNullOrEmpty(prefabData.SortPath) == false)
 			{
-				var Parent = Matrix.MetaTileMap.ObjectLayer.transform;
+				Transform Parent = null;
+				Parent = Matrix.MetaTileMap.ObjectLayer.transform;
```

Comment:
why?

Reply:
was debuging and trying different ways so it ended being the old approach

Reply:
ok, if you're going to make any changes to the code again; just put it back to its one line state.